src/processing/vFramer/include/vDraw.h

```diff
@@ -125,6 +131,34 @@ class accDraw : public vDraw {
 
 };
 
+class rasterDraw : public vDraw {
+
+protected:
+    #define XLIMIT 304          //array needs constants
+    #define YLIMIT 240
+
+    double pixelStorage[YLIMIT][XLIMIT];
+    int Xlimit = XLIMIT;
+    double Ylimit = YLIMIT;
```

Comment:
Xlimit and Ylimit are provided in the base-class and should not be replicated here